tests/qt_types_standalone/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ set(CRATE qt_types_standalone)
 cxx_qt_import_crate(
     MANIFEST_PATH rust/Cargo.toml
     CRATES ${CRATE}
-    QT_MODULES Qt::Core Qt::Gui Qt::Qml
+    QT_MODULES Qt::Core Qt::Gui Qt::Widgets Qt::Qml
```

Comment:
Could try adding QuickControls2 here which might help with the order, as qt_quickcontrols is enabled in cxx-qt-lib. But this ordering issue is still painful when it happens :-/